src/vaadin-grid-column.html

```diff
@@ -237,9 +237,7 @@
     }
 
     _selectFirstTemplate(selector) {
-      return Array.prototype.filter.call(Polymer.dom(this).querySelectorAll(selector), el => {
-        return Polymer.dom(el).parentNode === this;
-      })[0];
+      return Array.from(Polymer.dom(this).queryDistributedElements(selector))[0];
```

Comment:
Isn't Polymer.dom removed from Pure P2?

Reply:
Intended to postpone the cleanup (there are more places where legacy APIs are used in grid), but yeah, I might as well do this now. Fixed.